test/integration/navigation_selection_test.rb

```diff
@@ -6,20 +6,16 @@ class NavigationSelectionTest < ActiveSupport::TestCase
   Rails.application.routes.default_url_options[:host] = "example.com"
 
   TESTED_CONTROLLERS = %w[
-    announcements
     consultations
     corporate_information_pages
     email_signup_information
     histories
     home
     latest
     ministerial_roles
-    news_articles
```

Comment:
It'd be good to have why these are being removed in the commit message (I'm don't actually know why these are being removed)

Reply:
The reason for removing that is the same document types were removed from the tested method in `application_helper.rb`: https://github.com/alphagov/whitehall/pull/6950/commits/46b4ac87b96faf54577dfd8aea12aee588aa23e9#diff-5f3fc5e3977d242572aa1d08551f5eb557de0ccaff30370838ee9df5386ea0da.

As far as I could see, these methods were not being used anywhere (except in tests) and the method referred to a route helper which was deleted in the same commit.

I'll update the commit message to include this information.

Reply:
Looking at this more closely, I think this code can be removed completely as it is unused.  I've raised https://github.com/alphagov/whitehall/pull/6971 to do that and will rebase this branch once merged.